### DISCUSSION TITLE
MongoDB support for "ISODate()" and possible other functions in MongoDB query syntax

### DISCUSSION THREAD

**Discussion by WalterOesch** [dbeaver/dbeaver#19364] [Q&A]
Hello 
Im using DBeaver lite to Query MongoDB Atlas databases. I use a mix between SQL like syntax and the official MonogDB  syntax. IMO, some more MongoDB queries are only achievable with the MongoDB syntax. For simple ad hoc queries I prefere DBeavers SQL style syntax. 

When using "ISODate()" in MongoDB syntax in DBeaver I get the error "ReferenceError: "ISODate" is not defined. (js#1)". Is there a way to provide DBeaver with additional JavaScript libraries?

Would be great if I was able to use DBeaver for more MongoDB specific queries too.

Best regards,
Walter Oesch

**Comment by LonwoLonwo** [comment]
Hello @WalterOesch 

Thank you for the question. Please direct it here for a faster answer: https://dbeaver.com/tickets

**Reply by WalterOesch** [reply]
Thank you @LonwoLonwo. I created a ticket.
